### ISSUE TITLE
Bundle without UI/Controls

### ISSUE THREAD

**Issue by arski** [videojs/video.js#6976]
Hi all, in your VideoJS 7 roadmap notes you mentioned a future where there would be a config param like `VJS_CONTROLS=false` where one could bundle videoJS without all the UI elements. I'm seeing somewhat related threads like https://github.com/videojs/video.js/issues/6166 and https://github.com/videojs/video.js/issues/6012 - do you by any chance have any timeline on having a UI-free bundle? Effectively the tech alone and no UI controls?

Thanks

**Comment by gkatsev** [comment]
Unfortunately, we haven't had a chance to do that.

**Comment by arski** [comment]
I see, is it still planned?

**Comment by gkatsev** [comment]
Yes, it's something we want to provide but right now our highest priority is low latency playback.

**Comment by xxoo** [comment]
Is there any update for now?
